gateway/scaling/function_cache_test.go

```diff
@@ -81,7 +81,7 @@ func Test_CacheFunctionExists(t *testing.T) {
 
 	cache := FunctionCache{
 		Cache:  make(map[string]*FunctionMeta),
-		Expiry: time.Millisecond * 10,
+		Expiry: time.Millisecond * 100,
```

Comment:
Why did this change?

Reply:
due to the reduced speed because of emulation the cache was expiring before it got round to checking it had not expired

Reply:
:thinking: interesting. Thanks. Can you add that note as a commit above the line?

```
100ms - minimum required for the speed of this test during GitHub Actions
```